sort -V is only needed by the find_lib function, which doesn't work on
darwin anyway.

--- configure.orig	2022-10-05 10:48:35.000000000 +1100
+++ configure	2022-10-06 01:25:12.000000000 +1100
@@ -17950,10 +17950,16 @@
 done
 test -n "$SORT" || SORT="false"
 
-if ! "$SORT" -V </dev/null >/dev/null
-then :
-  as_fn_error $? "GNU sort(1) is required" "$LINENO" 5
-fi
+case $host_os in
+    darwin*)
+        ;;
+    *)
+        if ! "$SORT" -V </dev/null >/dev/null
+        then :
+          as_fn_error $? "GNU sort(1) is required" "$LINENO" 5
+        fi
+        ;;
+esac
 
 # Check whether --enable-largefile was given.
 if test ${enable_largefile+y}
